### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,27 @@
 * `npm install -g bower jshint gulp forever tsd typescript-formatter bower-installer` to install global dependencies
 * `bower install` will install client dependencies
 * `tsd update` will update typings folder with ambient files
-* Copy `config/localSettings.example.ts` to your own copy of `localSettings.ts` and add your eth0 address as the host, set a port
-* run `npm run dev` to start server and watch files
+* Copy `config/localSettings.example.ts` to your own copy of `localSettings.ts` and set the mediawikiHost, port and wikiFallback.
+    The mediawikiHost should be set to you're devbox name without 'dev-' prefix.
+    WikiFallback is useful but is not obligatory.
```
Comment (rogatty) [inline README.md]: I would use \` to mark all the property names (`mediawikiHost`, `port`, `wikiFallback`).

File: README.md [hunk 2]
```diff
@@ -7,15 +7,27 @@
...
-`127.0.0.1    lastofus.wikia-dev.local`
+    var localSettings = baseLocalSettings.getSettings({
+        wikiFallback: 'mediawiki119',
+        mediawikiHost: 'joe' #for dev-joe
+        port: 8000
+    });
+
+    export = localSettings;
+```
+* run `npm run dev` to start server and watch files
```
Comment (rogatty) [inline README.md]: How about capital letter so it's consistent with the previous bullet?

Comment (rogatty) [inline README.md]: Or just remove "run", it's not present in the previous bullets.

File: README.md [hunk 3]
```diff
@@ -7,15 +7,27 @@
...
+* Copy `config/localSettings.example.ts` to your own copy of `localSettings.ts` and set the mediawikiHost, port and wikiFallback.
+    The mediawikiHost should be set to you're devbox name without 'dev-' prefix.
+    WikiFallback is useful but is not obligatory.
+
+    File should look like this:
+``` javascript
+    import baseLocalSettings = require('./localSettings.base');
+    import Utils = require('../server/lib/Utils');
 
-## Setting up hosts
```
Comment (bognix) [inline README.md]: I removed "setting up hosts" section. I didn't set up hosts and mercury was working for me.

Comment (rogatty) [inline README.md]: Yup, it's from pre-xip.io era.

**Comment by bognix** [comment]
I removed "Set up the hosts" section because it wasn't needed if you're accessing Mercury via http://muppet.127.0.0.1.xip.io:8000/wiki/Gonzo .

**Comment by rogatty** [comment]
LGTM

**Pull request merged**